content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -11,11 +11,25 @@ title: QuestionnaireResponse | R4 API
 
 ## Overview
 
-The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. Only answer value types of String, Quantity, and Coding are supported. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history, which is a type of Observation, is supported. To find the corresponding social history questionnaire, first search QuestionnaireResponse by patient. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. Take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original Questionnaire.
 
 The following fields are returned if valued:
 
 * [Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Questionnaire](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status){:target="_blank"}
+* [Subject](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.encounter){:target="_blank"}
```

Comment:
i dont think we're supporting encounter?

Reply:
that also seemed weird to me but it was on the functional overview - https://jira2.cerner.com/browse/OPENSVC-19533

Reply:
Yeah i think when that was filled out we were still supporting it. Its greyed out on the wiki though, and were not currently returning it

Reply:
Removed: https://github.com/cerner/fhir.cerner.com/pull/669/commits/95b74ad7eacd678ba20c8069d315f3c02a32b8ab